### ISSUE TITLE
Installing seemingly unsupported Qt6 32-bit results in exception

### ISSUE THREAD

**Issue by altendky** [miurahr/aqtinstall#206]
I do not expect this to work since I think that Qt6 just doesn't support 32-bit MSVC on Windows, but I do expect it to provide a helpful error instead of a traceback.  As always, I very much appreciate this project.  Thank you.

```console
$ venv/bin/aqt install -O qt 6.0.0 windows desktop win32_msvc2019
2021-02-02 16:51:37,982 - aqt - INFO - aqtinstall(aqt) v1.0.0b3 on Python 3.9.0 [CPython GCC 9.3.0]
2021-02-02 16:51:39,014 - aqt - ERROR - Specified packages are not found while parsing XML of package information!
2021-02-02 16:51:39,015 - aqt - DEBUG - <Updates>
 <ApplicationName>{AnyApplication}</ApplicationName>
 <ApplicationVersion>1.0.0</ApplicationVersion>
 <Checksum>true</Checksum>
 <PackageUpdate>
  <Name>qt.qt6.600.debug_info</Name>
  <DisplayName>Qt Debug Information Files</DisplayName>
  <Description>Qt 6.0.0 debug information files. Requires selection of target binary like gcc or MSVC.</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Default>false</Default>
  <AutoDependOn/>
  <Script>installscript.qs</Script>
  <SortingPriority>2</SortingPriority>
  <DownloadableArchives/>
  <UpdateFile UncompressedSize="0" OS="Any" CompressedSize="0"/>
  <SHA1>cf804784bc3d43a4d54a2dbf23ed6ad8129b1086</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.debug_info.win64_mingw81</Name>
  <DisplayName>Desktop MinGW 8.1.0 64-bit debug information files</DisplayName>
  <Description>Qt 6.0.0 debug information files for Desktop MinGW 8.1.0 64-bit</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.debug_info, qt.qt6.600.win64_mingw81</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_mingw81</Dependencies>
  <Virtual>true</Virtual>
  <SortingPriority>2</SortingPriority>
  <DownloadableArchives>qtbase-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qtsvg-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qtdeclarative-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qttools-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qtquickcontrols2-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qt5compat-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qtshadertools-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qtquick3d-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z, qtquicktimeline-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64-debug-symbols.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="5393276681" OS="Any" CompressedSize="803972908"/>
  <SHA1>4d8643d5e6ebda4467126541ab5bd1673fac21cf</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.debug_info.win64_msvc2019_64</Name>
  <DisplayName>Desktop MSVC 2019 64-bit debug information files</DisplayName>
  <Description>Qt 6.0.0 debug information files for Desktop MSVC 2019 64-bit</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.debug_info, qt.qt6.600.win64_msvc2019_64</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_msvc2019_64</Dependencies>
  <Virtual>true</Virtual>
  <SortingPriority>2</SortingPriority>
  <DownloadableArchives>qtbase-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qtsvg-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qtdeclarative-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qttools-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qtquickcontrols2-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qt5compat-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qtshadertools-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qtquick3d-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z, qtquicktimeline-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64-debug-symbols.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="2466541928" OS="Any" CompressedSize="332165070"/>
  <SHA1>082e172c8cf8c867b01af93397bee20e98094f11</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qt5compat</Name>
  <DisplayName>Qt 5 Compatibility Module</DisplayName>
  <Description>Qt 5 Compatibility Module Prebuilt Components for Qt 6.0.0. Qt 5 Compatibility Module allows to continue using some dedicated functionality which has been removed in Qt 6.&lt;br>&lt;br>This component is available under commercial licenses from The Qt Company, or under GPL v3. For open source use, please note the additional requirements compared to LGPL v3.</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Default>false</Default>
  <SortingPriority>9</SortingPriority>
  <Dependencies>qt.qt6.600.doc.qt5compat, qt.qt6.600.examples.qt5compat</Dependencies>
  <AutoDependOn/>
  <Script>installscript.qs</Script>
  <DownloadableArchives/>
  <UpdateFile UncompressedSize="0" OS="Any" CompressedSize="0"/>
  <SHA1>45adf0c70a77dbe6e6d966ac338579804ffc5dca</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qt5compat.win64_mingw81</Name>
  <DisplayName>Qt 5 Compatibility Module for MinGW 8.1.0 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qt5compat, qt.qt6.600.win64_mingw81</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_mingw81</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qt5compat-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="1365840" OS="Any" CompressedSize="420103"/>
  <SHA1>beb085a042c5dee8238ebcdc8036ac93959fc0c1</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qt5compat.win64_msvc2019_64</Name>
  <DisplayName> Qt 5 Compatibility Module for MSVC 2019 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qt5compat, qt.qt6.600.win64_msvc2019_64</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_msvc2019_64</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qt5compat-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="3254997" OS="Any" CompressedSize="636663"/>
  <SHA1>924c60c882ff61d549ca35442805e7c4969e3245</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtquick3d</Name>
  <DisplayName>Qt Quick 3D</DisplayName>
  <Description>Qt Quick 3D provides high-level 3D API for Qt Quick.&lt;br>&lt;br>This component is available under commercial licenses from The Qt Company, or under GPL v3. For open source use, please note the additional requirements compared to LGPL v3.</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Default>false</Default>
  <SortingPriority>89</SortingPriority>
  <Dependencies>qt.qt6.600.doc.qtquick3d, qt.qt6.600.examples.qtquick3d, qt.qt6.600.qtshadertools, qt.qt6.600.qtquicktimeline</Dependencies>
  <AutoDependOn/>
  <Script>installscript.qs</Script>
  <DownloadableArchives/>
  <UpdateFile UncompressedSize="0" OS="Any" CompressedSize="0"/>
  <SHA1>03ea0c401062d3957ebf359e255196cc7dd509a8</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtquick3d.win64_mingw81</Name>
  <DisplayName>Qt Quick 3D for MinGW 8.1.0 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qtquick3d, qt.qt6.600.win64_mingw81</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_mingw81</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qtquick3d-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="9103195" OS="Any" CompressedSize="1878143"/>
  <SHA1>a3425acaf4d13f0beb26e630185b4e5c1fb2993c</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtquick3d.win64_msvc2019_64</Name>
  <DisplayName>Qt Quick 3D for MSVC 2019 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qtquick3d, qt.qt6.600.win64_msvc2019_64</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_msvc2019_64</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qtquick3d-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="25170521" OS="Any" CompressedSize="3808193"/>
  <SHA1>9821f65c151b55bcb1973386fcc6bcdfc4729bff</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtquicktimeline</Name>
  <DisplayName>Qt Quick Timeline</DisplayName>
  <Description>The Qt Quick Timeline module enables keyframe-based animations and parameterization. It takes a tooling-friendly approach, and is therefore directly supported by Qt Design Studio and Qt Quick Designer that contain a timeline editor for creating keyframe based animations.&lt;br>&lt;br>This component is available under commercial licenses from The Qt Company, or under GPL v3. For open source use, please note the additional requirements compared to LGPL v3.</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Default>false</Default>
  <SortingPriority>0</SortingPriority>
  <Dependencies>qt.qt6.600.doc.qtquicktimeline, qt.qt6.600.examples.qtquicktimeline</Dependencies>
  <AutoDependOn/>
  <Script>installscript.qs</Script>
  <DownloadableArchives/>
  <UpdateFile UncompressedSize="0" OS="Any" CompressedSize="0"/>
  <SHA1>7d4a4d1cb3e4085bfbaae4eb3050222654041904</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtquicktimeline.win64_mingw81</Name>
  <DisplayName>Qt Quick Timeline for MinGW 8.1.0 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qtquicktimeline, qt.qt6.600.win64_mingw81</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_mingw81</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qtquicktimeline-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="120139" OS="Any" CompressedSize="36750"/>
  <SHA1>2d291b08bccbe272c9c2b1fe4611820215e072be</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtquicktimeline.win64_msvc2019_64</Name>
  <DisplayName>Qt Quick Timeline for MSVC 2019 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qtquicktimeline, qt.qt6.600.win64_msvc2019_64</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_msvc2019_64</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qtquicktimeline-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="517343" OS="Any" CompressedSize="87370"/>
  <SHA1>856d5689c40df9a53eb292385345a4500361db67</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtshadertools</Name>
  <DisplayName>Qt Shader Tools</DisplayName>
  <Description>Qt Shader conditioning tool Prebuilt Components for Qt 6.0.0.&lt;br>&lt;br>This component is available under commercial licenses from The Qt Company, or under GPL v3. For open source use, please note the additional requirements compared to LGPL v3.</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Default>false</Default>
  <SortingPriority>9</SortingPriority>
  <Dependencies>qt.qt6.600.doc.qtshadertools, qt.qt6.600.examples.qtshadertools</Dependencies>
  <AutoDependOn/>
  <Script>installscript.qs</Script>
  <DownloadableArchives/>
  <UpdateFile UncompressedSize="0" OS="Any" CompressedSize="0"/>
  <SHA1>3dfcd6143434648c2bdc792c273a52d6f9ee2041</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtshadertools.win64_mingw81</Name>
  <DisplayName>Qt Shader Tools for MinGW 8.1.0 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qtshadertools, qt.qt6.600.win64_mingw81</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_mingw81</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qtshadertools-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="3543381" OS="Any" CompressedSize="1049091"/>
  <SHA1>59dc1a54dd0caa659c57fec2adf44f46e1c17f7d</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.qtshadertools.win64_msvc2019_64</Name>
  <DisplayName>Qt Shader Tools for MSVC 2019 64-bit</DisplayName>
  <Description/>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <AutoDependOn>qt.qt6.600.qtshadertools, qt.qt6.600.win64_msvc2019_64</AutoDependOn>
  <Dependencies>qt.qt6.600.win64_msvc2019_64</Dependencies>
  <Virtual>true</Virtual>
  <Script>installscript.qs</Script>
  <SortingPriority/>
  <DownloadableArchives>qtshadertools-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="13366545" OS="Any" CompressedSize="2296607"/>
  <SHA1>978580a0dea79dc13d942fcf9d0ede97dc3a91eb</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.win64_mingw81</Name>
  <DisplayName>MinGW 8.1.0 64-bit</DisplayName>
  <Description>Qt 6.0.0 Prebuilt Components for MinGW 8.1.0 64-bit</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Dependencies>qt.tools.qtcreator, qt.tools.win64_mingw810, qt.qt6.600.doc, qt.qt6.600.examples</Dependencies>
  <AutoDependOn/>
  <Default>false</Default>
  <Script>installscript.qs</Script>
  <SortingPriority>590</SortingPriority>
  <DownloadableArchives>qtbase-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z, qtsvg-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z, qtdeclarative-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z, qttools-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z, qtquickcontrols2-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z, qttranslations-Windows-Windows_10-Mingw-Windows-Windows_10-X86_64.7z, x86_64-8.1.0-release-posix-seh-rt_v6-rev0-runtime.7z, d3dcompiler_47-x64.7z, opengl32sw-64-mesa_11_2_2-signed.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="361854356" OS="Any" CompressedSize="70473794"/>
  <SHA1>92d407f9a95bc4b2cb3e77acb440961e9560ca51</SHA1>
 </PackageUpdate>
 <PackageUpdate>
  <Name>qt.qt6.600.win64_msvc2019_64</Name>
  <DisplayName>MSVC 2019 64-bit</DisplayName>
  <Description>Qt 6.0.0 Prebuilt Components for MSVC 2019 64-bit</Description>
  <Version>6.0.0-0-202012051252</Version>
  <ReleaseDate>2020-12-05</ReleaseDate>
  <Dependencies>qt.tools.qtcreator, qt.qt6.600.doc, qt.qt6.600.examples</Dependencies>
  <AutoDependOn/>
  <Default>false</Default>
  <Script>installscript.qs</Script>
  <SortingPriority>590</SortingPriority>
  <DownloadableArchives>qtbase-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z, qtsvg-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z, qtdeclarative-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z, qttools-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z, qtquickcontrols2-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z, qttranslations-Windows-Windows_10-MSVC2019-Windows-Windows_10-X86_64.7z, d3dcompiler_47-x64.7z, opengl32sw-64-mesa_11_2_2-signed.7z</DownloadableArchives>
  <UpdateFile UncompressedSize="603384669" OS="Any" CompressedSize="94268352"/>
  <SHA1>bd99add95b865d35e136b1598416f91f67abf404</SHA1>
 </PackageUpdate>
 <SHA1>cb95567aab26b81501478ea02d0397b5b6fc3554</SHA1>
 <MetadataName>2020-12-05-1310_meta.7z</MetadataName>
</Updates>

Traceback (most recent call last):
  File "/home/altendky/repos/qt-applications/venv/bin/aqt", line 8, in <module>
    sys.exit(main())
  File "/home/altendky/repos/qt-applications/venv/lib/python3.9/site-packages/aqt/__init__.py", line 38, in main
    return cli.run()
  File "/home/altendky/repos/qt-applications/venv/lib/python3.9/site-packages/aqt/installer.py", line 465, in run
    return args.func(args)
  File "/home/altendky/repos/qt-applications/venv/lib/python3.9/site-packages/aqt/installer.py", line 231, in run_install
    finisher(target_config, base_dir, self.logger)
  File "/home/altendky/repos/qt-applications/venv/lib/python3.9/site-packages/aqt/installer.py", line 562, in finisher
    raise e
  File "/home/altendky/repos/qt-applications/venv/lib/python3.9/site-packages/aqt/installer.py", line 547, in finisher
    with open(os.path.join(base_dir, qt_version, arch_dir, 'bin', 'qt.conf'), 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'qt/6.0.0/msvc2019/bin/qt.conf'
```

**Comment by miurahr** [comment]
Debug message with XML file is intended behavior, but the  exception is not.
```
FileNotFoundError: [Errno 2] No such file or directory: 'qt/6.0.0/msvc2019/bin/qt.conf'
```
That is because when the error happened, it should skip `qt.conf` handling, but not.

**Comment by miurahr** [comment]
Fix is released as v1.0.0.

**Comment by altendky** [comment]
Just confirmed the clean output with a clear error.

```console
$ venv/bin/aqt install -O qt 6.0.0 windows desktop win32_msvc2019
2021-02-04 21:17:16,863 - aqt - INFO - aqtinstall(aqt) v1.0.0 on Python 3.9.0 [CPython GCC 9.3.0]
2021-02-04 21:17:17,673 - aqt - ERROR - Specified packages are not found while parsing XML of package information!
```

As always, thank you for the quick fix.
